tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -725,6 +725,12 @@ platform_tests/daemon/test_ledd.py::test_pmon_ledd_kill_and_start_status:
 #######################################
 #####    fwutil/test_fwutil.py    #####
 #######################################
+platform_tests/fwutil/test_fwutil.py:
+  skip:
+  reason: "Doesn't support other platforms except Mellanox"
+  conditions:
+      - "asic_type not in ['mellanox']"
```

Comment:
we should ask other platform to support fwutil

Reply:
@prgeor Raised https://github.com/sonic-net/sonic-mgmt/issues/7811 to track it and will add this issue in condition.